README.md

```diff
@@ -2,13 +2,13 @@
 
 Basic Test Page for sending Audio to a websocket server from the browser
 
-This is a single HTML page that uses web audio and getUserMedia to create a client connection to a websocket server that sends and recieved audio, its useful for testing out websocket servers that are consuming the Nexmo websocket API without making the call from Nexmo.
+This is a single HTML page that uses web audio and getUserMedia to create a client connection to a websocket server that sends and receives audio, it's useful for testing out websocket servers that are consuming the Nexmo websocket API without making the call from Nexmo.
```

Comment:
This is a single HTML page that uses web audio and getUserMedia to create a client connection to a WebSocket server that sends and receives audio; it's useful for testing out WebSocket servers that are consuming the Nexmo WebSocket API without making the call from Nexmo.